packages/design-system/src/styles/components/_Choice.scss

```diff
@@ -243,7 +243,7 @@ label,
   border-left: $spacer-half solid $choice__border-color--left;
   flex: 1 1 100%;
   margin-bottom: $spacer-2;
-  margin-left: calc(calc(#{$choice__size} * 0.5) - calc(#{$spacer-half} * 0.5));
```

Comment:
When trying to test this, I realized that we don't use the `ds-c-choice__checkedChild` in our stories. It wasn't until I read the guidance docs that learned it was something we ask our users to add to their `checkedChildren` elements. While I don't love that from a usability perspective, I think we should probably start using it in our stories for now. Right now the visual regression tests aren't going to help us validate this change.

Reply:
The reason I was testing was that I wanted to visually see why we specifically need a 2px shift. Looking at our example...

<img width="155" alt="Screen Shot 2022-12-01 at 1 21 23 PM" src="https://user-images.githubusercontent.com/7595652/205161766-521d113d-f7d1-4e4a-8446-5ccc16c77cfe.png">

...I think it's actually because of the border on alerts and [healthcare.gov insets](https://design.cms.gov/components/inset?theme=healthcare)! This seems like a weird implementation detail and makes me think that in the future we actually want to work on making this part of the inset class and work on migrating it into the core design system. In other words, this generic class that we tell people to apply to all checked children seems to be more specifically meant for checked children that have a `4px`-wide border that we need to center vertically with the middle of the checkbox or radio button!

CC @line47, @brentschneider

Reply:
For now, @forrestbaer, could you add a comment above this line about where the `2px` comes from? (You can also convey that we're not 100% certain on the origin. I'm pretty sure it has nothing to do with `$spacer-half` though 😭)

Reply:
Could we set up some time to do a demo on this. It would help me understand what's going on here and give proper feedback.

Reply:
@line47, I moved this particular issue into https://github.com/CMSgov/design-system/pull/2285 and its associated ticket